README.md

```diff
@@ -19,7 +19,6 @@ documentation].
 
 | Release | Release Date |      EOL      |
 |:-------:|:------------:|:-------------:|
-|   v1.7  | Mar 22, 2022 |    Oct 2022   |
```

Comment:
I believe per the preceding paragraph we should also add the upcoming release (v1.11) to the table. I believe we would tentatively be targeting January 17th with EOL Oct 2023.